woocommerce/woocommerce
# Remove jetpack-boost

* Removed use of jetpack-boost -- we no longer run the experiment
* Jetpack should be considered as selected if it is already installed

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Related to https://github.com/woocommerce/woocommerce/pull/49452

This PR removes `jetpack-boost` related logics as we no longer run the experiment.

It also fixes a situation where Jetpack is not considered selected when it is already installed. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new JN site with this branch.
2. Go through the core profiler.
3. On the extensions page, click on `Continue`. We don't need to select Jetpack as it is already installed on JN sites.
4. You should be redirected to Jetpack connect page.
5. Confirm the URL has `plugin_name=jetpack-ai`



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance



-   [] Minor

#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Remove `jetpack-boost` as we no longer run the experiment.

</details>
